pages/linux/qmrestore.md

```diff
@@ -1,8 +1,20 @@
 # qmrestore
 
-> Restore QemuServer vzdump Backups.
+> Restore QemuServer vzdump backups.
 > More information: <https://pve.proxmox.com/pve-docs/qmrestore.1.html>.
 
-- Restore KVM-based virtual machine to local storage:
+- Restore virtual machine from given backup file on the original storage:
 
-`qmrestore {{/var/lib/vz/dump/backup_file.vma.lzo}} {{vm_id}} --storage {{local}}`
+`qmrestore {{path/to/vzdump-qemu-100.vma.lzo}} {{100}}`
```

Comment:
What means `vz`? we use `snake_case` for placeholders, this file should follow that case
```suggestion
`qmrestore {{path/to/vzdump_qemu_100.vma.lzo}} {{100}}`
```

Reply:
This is indeed not a simple placeholder.

`qmrestore` as a command seems to stem exclusively from Proxmox.

Proxmox VE/Backup Server vzdump backup files all begin with `vzdump-{{guest_type}}-{{vm_id}}`.

Since `qmrestore` is for QEMU guest types and seems to exclusively exist in the Proxmox ecosystem, you will exclusively encounter vzdump backup files beginning with  `vzdump-qemu-{{vm_id}}` in context of `qmrestore`

Unless you created the backup file outside of the Proxmox ecosystem or renamed it deliberately, this is exactly how the file will be named like.

Changing the hyphens to underscore will remove that reflection.

If you still want me to do this I will happily commit your suggestion.